types/stylelint/index.d.ts

```diff
@@ -634,9 +635,19 @@ declare namespace stylelint {
 		 * @internal
 		 * WARNING: Don't use this feature now. It may change in the future.
 		 */
-		fix?: () => void | never | Range;
+		fix?: (apply: boolean) => void | never | EditInfo;
 	};
 
+	type EditInfo =
+		| {
+				range?: Range;
+				text: string;
+		  }
+		| {
+				range: Range[];
+				text: string[];
+		  };
```

Comment:
@romainmenke Should I remove this and `| Range[]`/`| string[]` from `FixerData`?
This commit illustrate your proposal.

Reply:
>  Should I remove this and | Range[]/| string[] from FixerData?

Yes :)
I don't we think we need it at this time and anything that makes the POC simpler is always nice :)